home/service/search_tag_fetcher.py

```diff
@@ -0,0 +1,24 @@
+from django.core.cache import cache
+
+from .base import GenericService
+
+
+class SearchTagFetcher(GenericService):
+    def __init__(self):
+        self.client = self._get_catalogue_client()
+        self.cache_key = "search_tags"
+        self.cache_timeout_seconds = 5
```

Comment:
If this is a reasonably static list could we cache it for longer ?

Reply:
yeah we can - this just matches the other cache on domains - but they could both probably be increased